handlers.py

```diff
@@ -62,15 +69,49 @@ def handle(self, bot, update):
             else:
                 definition.content = update_text
             self.definitionRepo.persist(definition)
-            logger.debug("saved definition for %s", term)
+            logger.debug("Saved definition for %s", term)
             reply("Your definition for term '{}' has been saved".format(term))
             return
+        else:
+            reply("Please provide a new definition for term '{}'".format(term))
+
+    def handle_wtf(self, bot, update):
+        text = update.message.text
+        reply = update.message.reply_text
+        logger.debug("/wtf received: %s", text)
+        parts = text.split(" ", 2)
+
+        if len(parts) <= 1:
+            reply("Please provide a term for lookup.")
+            return
+
+        term = parts[1]
+        definition = self.definitionRepo.findByTerm(term)
 
         if definition is None:
             reply("I do not have any info for the term: '{}'".format(term))
         else:
             reply(("Term '{}' means: {}".format(definition.term, definition.content)))
```

Comment:
why double parentheses?

Reply:
I thought it's Python standard. Is there a PEP for this?

Reply:
this line isn't really different from the one above, you pass a string into `reply`. But you pass it inside of `()`

Reply:
Oh sorry, I was reading "quotes" not "parentheses" all the time